examples/images.jl

```diff
@@ -4,32 +4,28 @@ using SnoopCompile
 # This only needs to be run once (to generate "/tmp/images_compiles.csv")
 
 SnoopCompile.@snoop "/tmp/images_compiles.csv" begin
-    include(Pkg.dir("Images", "test","runtests.jl"))
+    include(Pkg.dir("Images", "test", "runtests.jl"))
 end
 
 ### Parse the compiles and generate precompilation scripts
 # This can be run repeatedly
 
-# IMPORTANT: we must have the module(s) defined for the parcelation
-# step, otherwise we will get no precompiles for the Images module
-using Images
-
 data = SnoopCompile.read("/tmp/images_compiles.csv")
 
 # The Images tests are run inside a module ImagesTest, so all
 # the precompiles get credited to ImagesTest. Credit them to Images instead.
-subst = Dict("ImagesTests"=>"Images")
+subst = ["ImagesTests"=>"Images", ]
```

Comment:
(Images no longer uses a module for its tests, but this is still useful as a demo.)